.circleci/config.yml

```diff
@@ -53,12 +59,22 @@ workflows:
           <<: *common_filters
           requires:
             - build
+      - regression-integration-tests:
+          filters:
+            branches:
+              only:
+                - master
+                - /^release.*$/
+          requires:
+            - build
 jobs:
   regression-integration-tests:
-    executor: integration_test_exec
+    executor: machine_integration_test_exec
```

Comment:
Confirm that machine_integration_test_exec means that only this job incurs the extra cost of running within a VM

Reply:
Yes, only the `regression-integration-tests` job will run in an Ubuntu VM